libnl-route-3.sym

```diff
@@ -1312,4 +1312,5 @@ global:
 	rtnl_link_bond_set_hashing_type;
 	rtnl_link_bond_set_miimon;
 	rtnl_link_bond_set_min_links;
+	rtnl_link_can_get_device_stats;
```

Comment:
please squash this commit in the previous.

the previous commit is incomplete/wrong, without this follow-up fix. No need to merge the change in two parts.